generators/ci-cd/templates/.gitlab-ci.yml.ejs

```diff
@@ -20,6 +20,18 @@
 image: jhipster/jhipster:v<%= jhipsterVersion %>
 <%_ } _%>
 
+<%_ if (prodDatabaseType === 'cassandra' || prodDatabaseType === 'redis' || prodDatabaseType === 'couchbase' || prodDatabaseType === 'neo4j') { _%>
+# DinD service is required for Testcontainers
+services:
```

Comment:
I think we should guard it also with `<%_ if (insideDocker) { _%>` as DinD service is usually only required when inside docker.

Reply:
Done :smile: